tests/core/config/models/test_bdk_datafeed_config.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 
 
-@pytest.fixture(params=["v1", 25, True])
+@pytest.fixture(params=["v1"])
```

Comment:
You should test with v2 and check it returns the DatafeedLoopV2

Reply:
This was actually to test default value of datafeed version is v1.

Reply:
I don't like to have much of logic in the model class. That's why I move the check to the `ServiceFactory` class which will instantiate the DatafeedLoop depending on the version presented in the config object.
https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/pull/124/files#diff-ef818d9162ea0611c768242c2a4089c29eae83aaa77223175f0f78dbb94af5c7R99-R110

Reply:
I added unittest for the `ServiceFactory` class.